src/components/content/catalog/services/details/ServiceDetail.tsx

```diff
@@ -73,6 +74,13 @@ function ServiceDetail({ serviceDetails }: { serviceDetails: ServiceTemplateDeta
                         showFor={ContactDetailsShowType.Catalog}
                     />
                 </Descriptions.Item>
+                <Descriptions.Item label='EULA'>
+                    {serviceDetails.eula ? (
+                        <AgreementText eula={serviceDetails.eula} />
+                    ) : (
+                        <Tag color='processing'>Not Provided</Tag>
```

Comment:
we do not need a tag here. It looks a bit odd. 
We can just but a simple text as "Not Provided"